server/rosetta/go.mod

```diff
@@ -0,0 +1,114 @@
+module github.com/cosmos/cosmos-sdk/rosetta
```

Comment:
should we move this into a new tools directory and then move cosmovisor to it? @julienrbrt @alexanderbez

Reply:
Then go mod name should be `cosmossdk.io/tools/rosetta`.

Reply:
So should we move the entire rosetta dir into a `tools` directory ?

Reply:
Yes, the go mod name must be changed as well.

Reply:
done!